pkg/codeowners/codeowners.go

```diff
@@ -0,0 +1,70 @@
+package codeowners
+
+import (
+	"os"
+
+	"github.com/hairyhenderson/go-codeowners"
+)
+
+type Codeowners struct {
+	codeowners *codeowners.Codeowners
+}
+
+func NewCodeowners(path string) (*Codeowners, error) {
+	// open filesystem rooted at current working directory
+	fsys := os.DirFS(path)
+
+	c, err := codeowners.FromFileWithFS(fsys, "CODEOWNERS")
+	if err != nil {
+		return nil, err
+	}
+
+	return &Codeowners{
+		codeowners: c,
+	}, nil
+}
+
+func (c *Codeowners) GetOwners(path string) (string, bool) {
+	if c == nil || c.codeowners == nil {
+		return "", false
+	}
+
+	owners := c.codeowners.Owners(path)
+	if len(owners) == 0 {
+		return "", false
+	}
+
+	ownerMap := map[string]string{
```

Comment:
Can't we find a way to externalize this as a config or something else? 

As it is like that, it can not be reused outside Qonto 😢

Reply:
you're perfectly right; I will think about a better place where to save this mapping;
maybe @mcorbin has some idea? I think we might save the mapping to a file in the Qonto's standard-insights deployment